rcl/src/rcl/guard_condition.c

```diff
@@ -133,7 +133,7 @@ rcl_guard_condition_get_default_options()
 }
 
 rcl_ret_t
-rcl_trigger_guard_condition(const rcl_guard_condition_t * guard_condition)
```

Comment:
These changes seem to undo recent commits to master. I think you need to make sure your rebase worked correctly...